harvester/products/sources/anatomy_tool.py

```diff
@@ -0,0 +1,245 @@
+            if hasattr(author, "fn"):
+                authors.append({
+                    "name": author.fn.value.strip(),
+                    "email": None,
+                    "external_id": None,
+                    "dai": None,
+                    "orcid": None,
+                    "isni": None,
+                })
+        return authors
+
+    @classmethod
+    def get_provider(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> dict[str, None | str]:
+        return {
+            "ror": None,
+            "external_id": None,
+            "slug": "anatomy_tool",
+            "name": "AnatomyTOOL"
+        }
+
+    @classmethod
+    def get_organizations(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> dict[str, list | dict[str, None | str]]:
+        root = cls.get_provider(soup, el)
+        root["type"] = "consortium"
+        return {
+            "root": root,
+            "departments": [],
+            "associates": []
+        }
+
+    @classmethod
+    def get_publishers(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> list[str]:
+        return ["AnatomyTOOL"]
+
+    @classmethod
+    def get_consortium(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> str:
+        return "AnatomyTOOL"
+
+    @classmethod
+    def get_publisher_date(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> None | str:
+        publisher = el.find(string='Created')
+        if not publisher:
+            return
+        contribution = publisher.find_parent('contribute')
+        if not contribution:
+            return
+        datetime = contribution.find('datetime')
+        if not datetime:
+            return
+        datetime = date_parser(datetime.text)
+        return datetime.strftime("%Y-%m-%d")
+
+    @classmethod
+    def get_publisher_year(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> None | int:
+        publisher_date = cls.get_publisher_date(soup, el)
+        if publisher_date is None:
+            return
+        datetime = date_parser(publisher_date)
+        return datetime.year
+
+    @classmethod
+    def get_lom_educational_levels(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> list[str]:
+        return ["HBO", "WO"]
+
+    @classmethod
+    def get_copyright_description(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> None | str:
+        node = el.find('rights')
+        if not node:
+            return
+        description = node.find('description')
+        return description.find('string').text.strip() if description else None
+
+    @classmethod
+    def get_learning_material_disciplines(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> list[str]:
+        return ["gezondheid"]
+
+
+ANATOMY_TOOL_EXTRACTION_OBJECTIVE = {
+    "@": AnatomyToolExtraction.get_oaipmh_records,
+    "external_id": AnatomyToolExtraction.get_oaipmh_external_id,
+    "set": lambda soup, el: f"oaipmh:{AnatomyToolExtraction.get_oaipmh_external_id(soup, el)}",
```

Comment:
Should be anatomy_tool: instead of oaipmh:. The Set name is used to uniquely identify a document (which could come in over different sets). The oaipmh doesn't say much about the Set that the document is a part of. It's only an indication of what protocol is being used, which downstream shouldn't matter (and doesn't afaik)